examples/open-records/open_records.md

```diff
@@ -1,6 +1,10 @@
 # Open records
 
-A record type that uses either the `{` and `}` delimiters or the `{|` and `|}` delimiters with a rest descriptor is considered open. They allow fields other than those specified. The type of unspecified fields is `anydata`. Open records belong to `map<anydata>`. Quoted keys can be used to specify fields that are not mentioned in the record type.
+A record type that uses either the `{` and `}` delimiters or the `{|` and `|}` delimiters with a rest descriptor is considered open. Open records allow fields other than those explicitly specified. 
+
+When the record is open due to using the  `{` and `}` delimiters, the expected type for any additional field is `anydata`. Such records belong to `map<anydata>`. 
```

Comment:
Shall we add a sentence to explain the expected type for additional fields if there is a rest type